tests/resources/Engine.py

```diff
@@ -2181,8 +2271,7 @@ def ctn_create_template_file(poller: int, typ: str, what: str, ids: list):
         what (str): A string. It depends on what type of template.
         ids (list): For each integer in this list, a template is defined.
     """
-    engine.ctn_create_template_file(poller, typ, what, ids)
-
+    engine.create_template_file(poller, typ, what, ids)
```

Comment:
is it normal you remove the ctn_ prefix?

Reply:
yes cause it refers to the staticmethod 
@staticmethod
    def create_template_file(poller: int, typ: str, what: str, ids):